Networking/Networking/Remote/CouponsRemote.swift

```diff
@@ -84,12 +88,34 @@ public final class CouponsRemote: Remote, CouponsRemoteProtocol {
 
         enqueue(request, mapper: mapper, completion: completion)
     }
+
+    /// Retrieves a `Coupon`.
+    ///
+    /// - Parameters:
+    ///     - siteID: Site for which we'll fetch the coupon.
+    ///     - couponID: ID of the Coupon that will be retrieved.
+    ///     - completion: Closure to be executed upon completion.
+    ///
+    public func retrieveCoupon(for siteID: Int64,
+                               couponID: Int64,
+                               completion: @escaping (Result<Coupon, Error>) -> Void) {
+        let request = JetpackRequest(wooApiVersion: .mark3,
+                                     method: .get,
+                                     siteID: siteID,
+                                     path: Path.coupons + "/\(couponID)",
+                                     parameters: [ParameterKey.force: true])
```

Comment:
just curious, what does the `force` parameter do?

Reply:
This is redundant, I've removed it in 123c37fd0167869ab69e947f9705b54649243185 👍